### DISCUSSION TITLE
App Structure

### DISCUSSION THREAD

**Discussion by ghost** [HuubMons/HuMo-genealogy#25] [Ideas]
A proposal for a new folder organization:
![humogen-proposition](https://github.com/HuubMons/HuMo-genealogy/assets/56915260/5ec42c2e-4d09-458b-bf79-6baf8acc2c37)

What do you think about it?

**Comment by HuubMons** [comment]
Yes, perfect! I don't mind changing files or directories to improve PHP structure.

**Comment by HuubMons** [comment]
Hi, Barcha,

I added a new "Version 7.0 development" in the discussion list. It wasn't possible to mention you in that topic.

**Reply by ghost** [reply]
No problem.
